node/runtime/pangolin/src/pallets/evm.rs

```diff
@@ -81,6 +82,13 @@ impl LatestMessageNoncer for ToPangoroMessageSender {
 	}
 }
 
+pub struct StorageFilter;
+impl StorageFilterT for StorageFilter {
+	fn allow(prefix: &[u8]) -> bool {
+		prefix != Twox128::hash(b"EVM") && prefix != Twox128::hash(b"Ethereum")
```

Comment:
the prefix is defined as "dvm", not like this EVM?
https://github.com/darwinia-network/darwinia-common/blob/main/frame/support/src/evm.rs#L75
and why we need put this Filter into runtime? I think it's a primitives function.

Reply:
> the prefix is defined as "dvm", not like this EVM?

Yes, We use the module name when calculating the storage key. After discussion with Echo and Denny,  we want to filter any state storage query related to EVM and Ethereum module to keep compatible with evm design.